mavedb_mapping/find_start_pos.py

```diff
@@ -0,0 +1,120 @@
+# Find start location in provided target sequence when start position is not first position of sequence
+import requests
+import Bio
+import re
+from Bio.Seq import Seq
+import pandas as pd
+import io
+from Bio.SeqUtils import seq1
+from Bio.Seq import Seq
+
+offset_within_ts = {}
+
+
+def validation_helper(protstring):
+    protstring = protstring[1:][:-1]
+    vs = protstring.split(";")
+    return vs
+
+
+def if_start_not_first(dat):
```

Comment:
not necessarily high priority, but it'd be great to refactor this method into a couple of different methods -- it's good practice to identify subsections in long methods and give them their own methods. For a long time, some people thought that you should enforce [strict 20 line limits](https://blog.knoldus.com/clean-code-robert-c-martins-way/) on methods, although that is probably too aggressive. Pylint, a popular Python code style checker, uses a default limit of [50 statements per method](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-statements.html), which is a good guideline to operate by IMO.